contracts/contracts/strategies/ThreePoolStrategy.sol

```diff
@@ -135,7 +148,9 @@ contract ThreePoolStrategy is InitializableAbstractStrategy {
             // in Gauge, unstake
             ICurveGauge(crvGaugeAddress).withdraw(withdrawPTokens);
         }
-        curvePool.remove_liquidity_one_coin(withdrawPTokens, poolCoinIndex, 0);
+        uint256 assetDecimals = Helpers.getDecimals(_asset);
+        uint256 minAssetAmount = _amount.scaleBy(int8(18 - assetDecimals));
```

Comment:
I think since we are calculating the max we could possibly withdraw, then dividing to get the amount we actually want, it'll always be higher than what we actually need anyway because of the slippage that comes into play when calculating the max we could withdraw. So I didn't think we'd need to do anything else. That make sense?

Reply:
ah yeah. thank you for the details. makes sense!